### PR TITLE
Adds resprites of existing guns + new sprites

### PR DESCRIPTION
**Pull request by schwicky-schwag**
## About the PR 
Adds some much-needed re-sprites for existing guns, including adding updated old gun sprites to match newer ones. Also contains a couple new sprites for guns to be used in the future, and new golden gun sprites.

Deagle firing sound modified from ["Desert_Eagle.wav"](https://freesound.org/people/seanmorrissey96/sounds/509429/) by [seanmorrissey96](https://freesound.org/people/seanmorrissey96/), used under [CC BY-NC](https://creativecommons.org/licenses/by-nc/3.0/)

Bullet casing sounds modified from [Pack: Bullet shells](https://freesound.org/people/CGEffex/packs/9828/?page=1#sound) by [CGEffex](https://freesound.org/people/CGEffex/), used under [CC BY](https://creativecommons.org/licenses/by/3.0/)

## Why's this needed? 
Some of the golden gun sprites are out of date, and none of them currently feature in-hand sprites. This PR adds updated sprites for old guns, gold sprites for newer guns, and in-hand sprites for all the golden guns.

## Changelog

```
(u)schwicky-schwag:
(*)Added a whole bunch of new gun sprites (with in-hands included) to replace old ones, and ones for coders to use in the future.
```

### CODE REVIEW

**Comment by Gannets** [comment]
Very pleased to see some of these in the game.

**Review by ZeWaka — CHANGES_REQUESTED** [review]
Looks like none of these, save the bling blaster and the shotgun inhand, are actually in use
I'm not so keen on adding a bunch of sprites that aren't even used anywhere.
Could you please set the various weapons to use the sprites?

**Comment by Tarmunora** [comment]
I'll be handling implementation

**Comment by Tarmunora** [comment]
@Gannets I poked the nukeop gun sprites to recenter them

**Comment by Tarmunora** [comment]
unused: g11, M16, bolt-actions for rifles, glocks, hi-point, bretta, deagle, unfolded spes, sawn-off, mac10

**Comment by schwicky-schwag** [comment]
> unused: g11, M16, bolt-actions for rifles, **glocks**, hi-point, bretta, deagle, unfolded spes, sawn-off, mac10

shouldn't the new glock sprites replace the old ones?

File: code/mob/living/carbon/human.dm [hunk 1]
```diff
@@ -1834,6 +1834,11 @@
 		hud.set_visible(hud.twohandr, 1)
 		hud.remove_item(I)
 		hud.add_object(I, HUD_LAYER+2, hud.layouts[hud.layout_style]["twohand"])
+		var/regex/offset = new(@"^icons/obj/(\d+)x\d+.dmi$")
+		if(offset.Find("[I.icon]"))
+			var/regex/off2 = new(@"^(CENTER[+-]\d:)(\d+)(.*)$")
```
Comment (ZeWaka) [inline code/mob/living/carbon/human.dm:1839]: could you document this cursed behemoth so people know what it does if they can't read through it?

File: code/mob/living/carbon/human.dm [hunk 2]
```diff
@@ -1880,6 +1885,11 @@
 		hud.set_visible(hud.rhand, 0)
 		hud.set_visible(hud.twohandl, 1)
 		hud.set_visible(hud.twohandr, 1)
+		var/regex/offset = new(@"^icons/obj/(\d+)x\d+.dmi$")
+		if(offset.Find("[I.icon]"))
+			var/regex/off2 = new(@"^(CENTER[+-]\d:)(\d+)(.*)$")
+			off2.Find("[I.screen_loc]")
+			I.screen_loc = "[off2.group[1]][text2num(off2.group[2])-(text2num(offset.group[1])-32)/2][off2.group[3]]"
```
Comment (ZeWaka) [inline code/mob/living/carbon/human.dm:1892]: ditto

**Review by ZeWaka — CHANGES_REQUESTED** [review]

**Review by ZeWaka — APPROVED** [review]
♥

**Pull request merged**